front-end/src/routes/export/containers/export.js

```diff
@@ -0,0 +1,12 @@
+import React,{Component} from 'react';
+
+
+export default class Export extends Component{
+    render(){
+        return (
+              <div>
+              exports
```

Comment:
Is this just a comment or temporary placement variable ?

Reply:
Temporary placement!